### DISCUSSION TITLE
Can Openradioss do 2D airblast numerical simulation by Euler mothod？

### DISCUSSION THREAD

**Discussion by tianlu6762** [OpenRadioss/OpenRadioss#1896] [Q&A]
Rt.

**Comment by MarianBulla** [comment]
Hi Tianlu,
thank you very much for your interest in OpenRadioss.
Yes, OpenRadioss can do 2D airblast simulaiton by using the Euler method.
To model this, you need the QUAD elements oriented in the Y-Z-plane:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/d00cc78c-15e6-4daa-ae1f-dd6ef1845a64)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/fd22ef28-3f98-445c-aa38-8414423d8d9c)

Hope it helps you ?
Thansk and best regards,
Marian

**Comment by tianlu6762** [comment]
Hi Marian,
2D airblast numerial simulation can bee done by using ale2d shell element in muli-materials in ls-dyna3d software, howerver, in OpenRadioss, 2D airblast can not been done by using purely shell element. Just as your say, 2D solid element means there is one element grid in x direction which is not a purely shell element.

**Reply by PaulAltair** [reply]
I'm not sure what distinction you are making? You cannot do ALE in shells sure, but the OpenRadioss solution is 2D in the example Marian shared, what do you mean by a grid in X?

**Comment by tianlu6762** [comment]
Hi sir,The model is created in y_z plane in Marian example shared , how many elements are the model in x axial direction？Best regardsTianlu发自我的手机-------- 原始邮件 --------发件人： Pauly ***@***.***>日期： 2023年9月24日周日 凌晨1:00收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Author ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] Can Openradioss do 2D airblast numerical simulation by Euler mothod？ (Discussion #1896)
I'm not sure what distinction you are making? You cannot do ALE in shells sure, but the OpenRadioss solution is 2D in the example Marian shared, what do you mean by a grid in X?

**Reply by PaulAltair** [reply]
there is only 1 element in the plane, it has no 'thickness' the elements are '2D solid' (they are described by only 4 nodes each) and represent a slice through a solid entity (2D analysis can be plane or axisymmetric)

**Comment by tianlu6762** [comment]
Mr.  Pauly,Thank you reply I am very happy to receive you information about 2d airblast by using euler method.Can you share 2D blast model example to me?Best regardsTianlu发自我的手机-------- 原始邮件 --------发件人： Pauly ***@***.***>日期： 2023年9月24日周日 凌晨1:00收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Author ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] Can Openradioss do 2D airblast numerical simulation by Euler mothod？ (Discussion #1896)
I'm not sure what distinction you are making? You cannot do ALE in shells sure, but the OpenRadioss solution is 2D in the example Marian shared, what do you mean by a grid in X?

**Comment by tianlu6762** [comment]
Hi Pauly,Would you please share the following example to me？Best regardsTianlu-------- 原始邮件 --------发件人： Pauly ***@***.***>日期： 2023年9月24日周日 凌晨1:00收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Author ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] Can Openradioss do 2D airblast numerical simulation by Euler mothod？ (Discussion #1896)
I'm not sure what distinction you are making? You cannot do ALE in shells sure, but the OpenRadioss solution is 2D in the example Marian shared, what do you mean by a grid in X?

**Comment by RevolT87** [comment]
Hi @tianlu6762,
thanks for your interest in simulating airblast. 
I recommand you to have a look at this article: https://community.altair.com/community/en/250kg-tnt-air-burst-at-3m-simulation?id=kb_article_view&sysparm_article=KB0122919 
You will find models & describing what you are looking for. 
regards,
Mathis.

**Comment by tianlu6762** [comment]
Would you please take some time to check my input file of 2D airblast numerical simulations？Best regards发自我的手机-------- 原始邮件 --------发件人： Pauly ***@***.***>日期： 2023年9月26日周二 下午4:54收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Mention ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] Can Openradioss do 2D airblast numerical simulation by Euler mothod？ (Discussion #1896)
there is only 1 element in the plane, it has no 'thickness' the elements are '2D solid' (they are described by only 4 nodes each) and represent a slice through a solid entity (2D analysis can be plane or axisymmetric)

**Reply by Schwoertzig** [reply]
Hi @tianlu6762 ,
yes I can have a look to your 2D blast modeling.
Could you please attach your input file or provide a link ?
Regards
Thierry
